packages/insea-connect-backend/src/main/java/ma/insea/connect/chat/group/GroupService.java

```diff
@@ -37,7 +39,10 @@ public Group saveGroup(GroupDTO groupDTO) {
             m.setIsAdmin(false);
             m.setJoiningDate(new java.sql.Date(System.currentTimeMillis()));
             membershipRepository.save(m);
-        }      
+        }
+        Membership m=membershipRepository.findById(new MembershipKey(group.getCreator(), group.getId())).get();
```

Comment:
I guess, It's possible to not have another request to the database, you can use the groupDTO object to create the membership and save the group at the end